src/app/shared/components/forms/custom-checkbox/custom-checkbox.component.ts

```diff
@@ -9,33 +9,37 @@ import {
 import { FormArray, FormControl, FormGroup } from '@angular/forms';
 import { JsonFormControls } from 'src/app/models/json-form-data.model';
 import isEqual from 'lodash.isequal';
```

Comment:
uninstall library if not being used any more

Reply:
lodash is using in select-prescription component, but i removed this from top of this file as not using it in this